python/mdvtools/auth/authutils.py

```
@@ -0,0 +1,359 @@
+            db.session.commit()
+
+            if is_admin:
+                # Assign all projects to this user as owner via UserProjectService
+                for project in Project.query.all():
+                    UserProjectService.add_or_update_user_project(
+                        user_id=db_user.id,
+                        project_id=project.id,
+                        is_owner=True
+                    )
+
+        logger.info(f"Synced users from Auth0.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)